openprescribing/pipeline/management/commands/pipeline_e2e_tests.py

```diff
@@ -0,0 +1,164 @@
+from distutils.dir_util import copy_tree
+import os
+import shutil
+
+from django.core.management import BaseCommand, CommandError
+from django.conf import settings
+from django.test import override_settings
+
+from django.core.management import call_command
+from django.db import connection
+
+from gcutils.bigquery import Client as BQClient, DATASETS
+from gcutils.storage import Client as StorageClient
+from frontend import bq_schemas as schemas
+from frontend.models import MeasureValue, MeasureGlobal
+from pipeline import runner
+
+
+e2e_path = os.path.join(settings.SITE_ROOT, 'pipeline', 'e2e-test-data')
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        if os.environ['DJANGO_SETTINGS_MODULE'] != 'openprescribing.settings.test':
+            raise CommandError('Command must run with test settings')
+
+        with override_settings(
+            PIPELINE_METADATA_DIR=os.path.join(
+                settings.SITE_ROOT,
+                'pipeline',
+                'metadata'),
+            PIPELINE_DATA_BASEDIR=os.path.join(e2e_path, 'data', ''),
+            PIPELINE_IMPORT_LOG_PATH=os.path.join(e2e_path, 'log.json'),
+        ):
+            run_end_to_end()
+
+
+def run_end_to_end():
+    print('BQ_NONCE: {}'.format(settings.BQ_NONCE))
+
+    num_measures = 56
```

Comment:
IIUC this number will change each time we add or remove a measure. It would be good to add a custom message to make this obvious following a failure at L80, e.g. "have measures been added or deleted? If so, change `num_measures` to match".